CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-*                                                               @simonz130 @Daniel-Sanche
+*                                                               @Daniel-Sanche @arbrown @minherz
 /.gitattributes                                                 @GoogleCloudPlatform/observability-devx
 /.gitignore                                                     @GoogleCloudPlatform/observability-devx
 /src/                                                           @simonz130 @GoogleCloudPlatform/observability-devx
```

Comment:
please, use @GoogleCloudPlatform/dee-observability team instead of distinct users or other @GoogleCLoudPlatform groups. I've created this group to host and maintain o11y activities for DEE.

Reply:
Ok, done.  We'll have to resolve the conflict when merging develop into main, but it's pretty straightforward what to do.